src/utils/README.md

```diff
@@ -0,0 +1,40 @@
+# Utils
+
+## resolveVisualEditorData
+
+This is a helper function to be used in transformProps within your template tsx files (ex.
+`location.tsx`).
+
+### Params
+
+| Name                       | Type                     |
+| -------------------------- | ------------------------ |
+| entityConfigurations       | VisualConfiguration[]    |
+| entityLayoutConfigurations | PagesLayout[]            |
+| siteLayoutConfigurations   | VisualLayout[]           |
+| templateName               | json string or undefined |
+
+### Usage
+
+```tsx
+export const transformProps = async (data) => {
+  const { document } = data;
+  const entityConfigurations = document.c_visualConfigurations ?? [];
+  const entityLayoutConfigurations = document.c_pages_layouts ?? [];
+  const siteLayoutConfigurations = document._site?.c_visualLayouts;
+  const visualEditorData = resolveVisualEditorData(
```

Comment:
I'm thinking we have this method take in a document and return the document, then people have to do even less.

Reply:
Will be fixed in another PR

Reply:
will dod in separate PR